client/src/modules/dashboard/Sponsors/graphql/queries.ts

```diff
@@ -0,0 +1,35 @@
+import { gql } from '@apollo/client';
+
+export const SPONSOR = gql`
+  query sponsor($sponsorId: Int!) {
+    sponsor(id: $sponsorId) {
+      id
+      name
+      website
+      logo_path
+      type
+    }
+  }
+`;
+
+export const createSponsor = gql`
+  mutation createSponsor($data: CreateSponsorInputs!) {
+    createSponsor(data: $data) {
+      name
+      website
+      logo_path
+      type
+    }
+  }
+`;
+
+export const updateSponsor = gql`
+  mutation UpdateSponsor($data: UpdateSponsorInputs!, $updateSponsorId: Int!) {
```

Comment:
```suggestion
  mutation updateSponsor($data: UpdateSponsorInputs!, $updateSponsorId: Int!) {
```

Doesn't matter, but the rest of the query names seem to use camel case

Reply:
@moT01 , I've fixed this.